src/ad-sizes.ts

```diff
@@ -83,9 +84,9 @@ type SizeKeys =
 	| 'outstreamDesktop'
 	| 'outstreamGoogleDesktop'
 	| 'outstreamMobile'
+	| 'picnic'
```

Comment:
Could we keep this just as the dimensions? As picnic might be one ad that runs but others could in the future I.e. Google Native / house ads for example.

Reply:
Sure! It looks like we give the dimension-based ones a name first. From a bit of googling should it be a [portrait interstitial](https://www.dimensions.com/element/mobile-portrait-interstitial-ad-320-x-480)?

Reply:
Okay TIL! Makes sense.